### ISSUE TITLE
Update PP endpoint to support receiving filters

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#17985]
We have two endpoints:

1. `/DotAjaxDirector/com.dotcms.publisher.ajax.RemotePublishAjaxAction/cmd/pushBundle`
2. `/DotAjaxDirector/com.dotcms.publisher.ajax.RemotePublishAjaxAction/cmd/publish`

They both receive a payload of something like:

```
assetIdentifier=855a2d72-f2f3-4169-8b04-ac5157c4380c&remotePublishDate=2020-02-12&
remotePublishTime=3-38&remotePublishExpireDate=2020-02-12&
remotePublishExpireTime=3-38&iWantTo=publish&whoToSend=bbb05294-d561-480e-
a64c-73bdbbc279a3&bundleName=&bundleSelect=&forcePush=false
```

Now we need to be able to pass a filter identifier.

Note: If force push is called programmatically, we still respect it and ignore the filters.

**Comment by hmoreras** [comment]
core-web PR when the endpoint is done: https://github.com/dotCMS/core-web/pull/1316

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/18428
PR EE: https://github.com/dotCMS/enterprise-2.x/pull/725

**Comment by bryanboza** [comment]
Fixed, tested on release 5-3.3 and now we are accepting the PP filters.
